packages/relay/src/lib/db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository.ts

```diff
@@ -102,6 +102,36 @@ export class IPAddressHbarSpendingPlanRepository {
     }
   }
 
+  /**
+   * Deletes all IP address spending plans from the cache.
+   *
+   * @param {RequestDetails} requestDetails - The request details for logging and tracking.
+   * @returns {Promise<void>} - A promise that resolves when all IP address spending plans are deleted.
+   */
+  async deleteAll(requestDetails: RequestDetails): Promise<void> {
+    const key = this.getKey('*');
+    const keys = await this.cache.keys(key, 'deleteAll', requestDetails);
```

Comment:
Let's try to use the `name` property of this particular method instead of a string. This way if the method name is ever updated, we won't have to worry about updating the string.

```suggestion
    const keys = await this.cache.keys(key, this.deleteAll.name, requestDetails);
```

Reply:
done, good point